app/models/state_file1099_g.rb

```diff
@@ -30,7 +30,7 @@
 #
 class StateFile1099G < ApplicationRecord
   belongs_to :intake, polymorphic: true
-  before_save :update_conditional_attributes
+  before_validation :update_conditional_attributes
```

Comment:
Was the intent here so that we it will pre-validate presence of the 3 attributes before updating trying to conditional_attributes? Do we need to validate mailing_apartment?

Reply:
The intent here is that the update to the conditional attributes happens before validation instead of after it but before save. This way, the conditional attributes get set, and then validation happens, so the values are valid. (Instead of validation happening and then the conditional attributes getting set, resulting in an invalid model)